conf/machine/include/imx-base.inc

```diff
@@ -34,7 +34,7 @@ UBOOT_ENTRYPOINT_vf = "0x80008000"
 
 PREFERRED_PROVIDER_virtual/xserver = "xserver-xorg"
 XSERVER_DRIVER                  = "xf86-video-fbdev"
-XSERVER_DRIVER_imxgpu2d         = "xf86-video-imx-vivante"
+XSERVER_DRIVER_imxgpu           = "xf86-video-imx-vivante"
```

Comment:
Hi Tom, I disagree with that change since xf86-video-imx-vivante requires libg2d:
https://github.com/Freescale/meta-freescale/blob/master/recipes-graphics/xorg-driver/xf86-video-imx-vivante_6.2.4.p1.6.bb#L12

Reply:
For instance i.MX8MQ has imxgpu but no g2d provider. But maybe we need to add something about dpu2d.

Reply:
Thanks, I deleted the patch. Note that for i.MX 8, we're not supporting X11 backend.